src/main/api/verifyEmailAddressByUserId.json

```diff
@@ -0,0 +1,22 @@
+{
+  "uri": "/api/user/verify-email",
+  "comments": [
+    "Administratively verify a user's email address. Use this method to bypass email verification for the user.",
+    "",
+    "The request body will contain the userId to be verified. This manual verification method requires an API key."
```

Comment:
> This manual verification method requires an API key.

If we are going to mention this at all, it should be probably be something like `An API key is required when sending the userId in the request body.`